CHANGELOG.md

```diff
@@ -18,6 +18,10 @@
 
   Description goes here (optional)
 
+  ([PR #1233](https://github.com/alphagov/govuk-frontend/pull/1233))
+
+- Add classes to action items in the summary list component
```

Comment:
Small issue here with the CHANGELOG being slightly out of order, also we like to acknowledge contributors in here too if you want to thank yourself.

I can sort this for you if you want, let me know.

Reply:
oops, ive just pushed a change for that

Reply:
oh - should i not have updated this file?

Reply:
I was not clear, the changelog entry is great, it's just you have the link to the pull request above the title which could be confusing when people read this back

It should be below it instead, if you compare the other entries.

Since these things are boring, I'm happy to do this for you, up to you.

Reply:
Looks good now thanks :)